documentation/tutorial.md

```diff
@@ -1,28 +1,37 @@
-# Charmed MongoDB tutorial:
-Ready to try out Charmed MongoDB? This tutorial will guide you through the steps to get Charmed MongoDB up and running. 
+# Charmed MongoDB tutorial
+The Charmed MongoDB Operator delivers automated operations management from day 0 to day 2 on the [MongoDB Community Edition](https://github.com/mongodb/mongo) document database. It is an open source, end-to-end, production-ready data platform on top of juju. As a first step this tutorial show you how to get Charmed MongoDB up and running, but the tutorial does not stop there. Through this tutorial you will learn a variety of operations, everything from adding replicas to advanced operations such as enabling TLS. Specifically this tutorial will cover:
+- Setting up your environment using LXD and Juju.
+- Deploying MongoDB using a single command.
+- Accessing the admin database directly.
+- Adding high availability with replication.
+- Changing the admin password.
+- Automatically creating users with relations. 
+- Enabling secure transactions with TLS.
 
-## Minimum requirements:
+
+## Minimum requirements
 Before we start be sure you have the following requirements:
 - Ubuntu 20.04(focal) or later.
 - 8GB of RAM.
 - 2 CPU threads.
 - At least 20GB of available storage.
 - Access to the internet for downloading the required snaps and charms.
 
-## Install and prepare LXD:
+
+## Install and prepare LXD
 The fastest, simplest way to get started with Charmed MongoDB is to set up a local LXD cloud. LXD is a system container and virtual machine manager; Charmed MongoDB will be run in one of these containers and managed by Juju. While this tutorial covers the basics of LXD, you can explore more LXD [here](https://linuxcontainers.org/lxd/getting-started-cli/). The first step on our journey is to install LXD. LXD is installed from a snap package:
```

Comment:
```suggestion
The fastest, simplest way to get started with Charmed MongoDB is to set up a local LXD cloud. LXD is a system container and virtual machine manager; Charmed MongoDB will be run in one of these containers and managed by Juju. While this tutorial covers the basics of LXD, you can [explore more LXD here](https://linuxcontainers.org/lxd/getting-started-cli/). The first step on our journey is to install LXD. LXD is installed from a snap package:
```

Reply:
Longer links are easier for people with screen readers and magnifiers to manage :)